src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final int DIVIDER = 2;
+    private double height;
+    private double firstSide;
+    private double secondSide;
+
+    public IsoscelesTrapezoid(String color, double height, double firstSide, double secondSide) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables for 'height', 'firstSide', and 'secondSide' and pass them on for better code readability.